relay-dynamic-config/src/defaults.rs

```diff
@@ -117,7 +117,7 @@ pub fn hardcoded_span_metrics() -> Vec<(GroupKey, Vec<MetricSpec>, Vec<TagMappin
     let is_db = RuleCondition::eq("span.sentry_tags.category", "db")
         & !RuleCondition::glob("span.op", DISABLED_DATABASES)
         // MongoDB queries are only allowed when `span.system` is set to `mongodb`.
-        & (RuleCondition::eq("span.system", "mongodb")
```

Comment:
@mjq turns out this condition had the wrong field, which we did not notice because `MONGODB_QUERIES` contained an invalid pattern.